packages-excludedocs.ks.in

```diff
@@ -12,11 +12,14 @@
 %packages --excludedocs
 @container-management
 @core
-@domain-client
 @hardware-support
 @headless-management
 @server-product
 @standard
+# The vdo package  stores license files in the docs folder
+# so they are still installed even if we tell DNF to
+# not install docs.
+-vdo
```

Comment:
This should be reported as a bug (see the %post section below). However, I think this test is too ambitious. Maybe, we could minimize the number of installed packages or check only a few directories in /usr/share/doc. We should verify that anaconda does its job, we don't have to verify that all packages respect the excludedocs flag.

Reply:
Agreed, I was thinking there must have been some reason for so many groups. I've updated the %packages section.